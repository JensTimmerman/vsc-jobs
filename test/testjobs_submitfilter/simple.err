Unable to determine clustername, using default delcatty (no PBS_DEFAULT)
