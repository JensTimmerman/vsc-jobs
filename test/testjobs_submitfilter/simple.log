^.*? WARNING\s+submitfilter.syslogger\s+MainThread\s+submitfilter\s+-\sno \[vp\]mem specified by user \w+. adding 4720302336.*$
