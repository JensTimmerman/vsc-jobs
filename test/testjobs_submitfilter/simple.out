#!/bin/bash
# No mail specified - added by submitfilter
#PBS -m n
# No vmem limit specified - added by submitfilter (server found: delcatty)
#PBS -l vmem=4897988864
hostname
